recipe/build-pkg.bat

```diff
@@ -1,7 +1,22 @@
+SetLocal EnableDelayedExpansion
+
+if "%cuda_compiler_version%"=="None" (
+    set FAISS_ENABLE_GPU="OFF"
+) else (
+    set FAISS_ENABLE_GPU="ON"
+
+    REM See more extensive comment in build-pkg.sh
+    REM TODO: Fix this in nvcc-feedstock or cmake-feedstock.
+    del %BUILD_PREFIX%\bin\nvcc
```

Comment:
The Windows wrapper does have a `.bat` extension, so it wasn't getting deleted,  CMake could find it, bad CUDA root found, etc....

```suggestion
    del %BUILD_PREFIX%\bin\nvcc.bat
```

Reply:
It's always obvious in hindsight, isn't it...
Thanks so much for debugging this - let's hope it works! :)